virttest/remote.py

```diff
@@ -1442,3 +1455,31 @@ def run_command(self, vm_ip, command, vm_user="root", runner=None,
             raise exceptions.TestFail("Failed to run '%s' in the VM: %s",
                                       cmd, detail)
         return ret
+
+    def cmd_output_safe(self, cmd, timeout=CMD_TIMEOUT):
+        """
+        Unify the interface for session.cmd_output_safe()
+        :param cmd: The command to execute
+        :param timeout: Ignored so far
+        :return: cmd_result.stdout_text
+        """
+        return self.run_command(cmd).stdout_text.strip()
+
+    def cmd_status(self, cmd, safe=True):
+        """
+        Unify the interface for session.cmd_status()
+        :param cmd: The command to be executed
+        :param safe: Ignored so far
+        :return: cmd_result.exit_status
+        """
+        return self.run_command(cmd).exit_status
+
+    def cmd_status_output(self, cmd, timeout=CMD_TIMEOUT):
+        """
+        Unify the interface for session.cmd_status_output()
+        :param cmd: The command to be executed
+        :param timeout: Ignored so far
+        :return: cmd_result.exit_status and cmd_result.stdout
+        """
+        return (self.run_command(cmd).exit_status,
+                self.run_command(cmd).stdout_text.strip())
```

Comment:
I think here should not do run_command twice, but once and return ret.exit_status and ret.stdout_text.strip()

Reply:
@kylazhang yeah, thanks and update.